azure-pipelines.yml

```diff
@@ -130,21 +134,17 @@ jobs:
           publishTestResults: true
           testRunTitle: '$(Agent.JobName) on $(Agent.OS)'
 
-  - job: Analysis
-    pool:
-      name: 00-OSIManaged-Containers
-      demands: Agent.OS -equals Windows_NT
-    steps:
-      - task: DotNetCoreCLI@2
-        displayName: 'Nuget restore'
-        inputs:
-          command: restore
-
-      - task: DotNetCoreCLI@2
-        displayName: 'Build'
-        inputs:
-          command: build
+  - template: '/miscellaneous/build_templates/code-analysis.yml@templates'
+    parameters:
+      buildSteps:
+        - task: DotNetCoreCLI@2
+          displayName: 'Nuget restore'
+          inputs:
+            command: restore
 
-      - template: '/miscellaneous/build_templates/binskim.yml@templates'
+        - task: DotNetCoreCLI@2
+          displayName: 'Build'
+          inputs:
+            command: build
 
-      - template: '/miscellaneous/build_templates/analysis.yml@templates'
+        - template: '/miscellaneous/build_templates/binskim.yml@templates'
```

Comment:
Sorry if I am missing something, but how come we run a binskim template in some builds and some we don't?

Reply:
depends on the language, binskwim works only on exes and dlls